src/app.js

```diff
@@ -6,6 +6,7 @@ const compression = require('compression');
 const bodyParser = require('body-parser');
 const path = require('path');
 var siofu = require("socketio-file-upload");
+var generator = require('./backend/generator.js');
```

Comment:
yes, I think the [node environment](http://eslint.org/docs/user-guide/configuring) should be set to true.